src/components/SearchDropdown/SearchDropdown.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC, useState } from "react";
+
+import Flex from "@/components/Flex";
+import Box from "@/components/Box";
+import Button from "@/components/Button";
+import OakLink from "@/components/OakLink";
+
+type SearchDropdownProps = {
+  dropdownTitle: string;
+  dropdownContent: DropdownContentItem[];
+};
+
+export type UnitContentItem = {
+  type: "unit";
+  unitTitle: string;
+  unitSlug: string;
+  programmeSlug: string;
+  examboardTitle?: string;
+  tierTitle?: string | null;
+};
+
+export type LessonContentItem = {
+  type: "lesson";
+  lessonTitle: string;
+  lessonSlug: string;
+  programmeSlug: string;
+  unitSlug: string;
+  examboardTitle?: string;
+  tierTitle?: string;
+};
+
+type DropdownContentItem = UnitContentItem | LessonContentItem;
+
+const SearchDropdown: FC<SearchDropdownProps> = ({
+  dropdownTitle,
+  dropdownContent,
+}) => {
+  const [isToggleOpen, setToggleOpen] = useState<boolean>(false);
+
+  return (
+    <Flex $flexDirection={"column"} $alignItems={"flex-start"}>
+      <Button
```

Comment:
I think this should be a[ new component ](https://www.figma.com/file/YcWQMMhHPVVmc47cHHEEAl/Oak-Design-Kit?node-id=3350%3A5976&mode=dev)instead of using the current button as the focus/hover states look different